### PR TITLE
Add section on spread operator

### PR DESCRIPTION
**Pull request by ysthakur**
Adding a section on the spread operator to the release notes for 0.88. Let me know if it needs more information or to be changed otherwise.

### CODE REVIEW

File: blog/2023-12-12-nushell_0_88_0.md [hunk 1]
```diff
@@ -80,6 +80,22 @@ Thanks to all the contributors below for helping us making the documentation of
...
+[#10598](https://github.com/nushell/nushell/issues/10598) asked for a spread operator, and [#11006](https://github.com/nushell/nushell/pull/11006)
+and [#11144](https://github.com/nushell/nushell/pull/11144/) implemented a spread operator in lists and records, respectively.
+The spread operator can help one avoid `append` (when used in lists) and `merge` (when used in records). Spreading arguments to commands has not been implemented yet.
+
+```nushell
+> let x = ["foo"]; [ 1 ...$x ...("foo" | split chars) ...[3 9] [] ] | to nuon
+[1, foo, f, o, o, 3, 9, []]
+> let x = { a: "foo" }; { ...$x y: "bar" ...{ x: 2 } } | to nuon
+{a: foo, y: bar, x: 2}
+```
```
Comment (amtoine) [inline blog/2023-12-12-nushell_0_88_0.md:94]: i would simply suggest splitting these into two blocks, putting the code on multiple lines, to hopefully make the snippets a bit easier to read :yum:
> suggestion comes after this line

for the example, let's create a list and a record
```nushell
let list = ["foo"]
let record = { a: "foo" }
```
- on lists
```nushell
[
    1,
    ...$x,
    ...("foo" | split chars),
    ...[3 9],
    []
] | to nuon
```
will give
```
[1, foo, f, o, o, 3, 9, []]
```
- on records
```
{
    ...$x,
    y: "bar",
    ...{ x: 2 }
} | to nuon
```
will give
```
{a: foo, y: bar, x: 2}
```

**Review by amtoine — COMMENTED** [review]
thanks @ysthakur for adding this for us :pray: 
much appreciated :blush: 

i just have a suggestion, tell me what you think about it :wink:

File: blog/2023-12-12-nushell_0_88_0.md (hunk 1 above)
Comment (ysthakur) [reply blog/2023-12-12-nushell_0_88_0.md]: Applied, with some minor changes, let me know what you think

Comment (amtoine) [reply blog/2023-12-12-nushell_0_88_0.md]: looks very nice :ok_hand:

File: blog/2023-12-12-nushell_0_88_0.md [hunk 2]
```diff
@@ -80,6 +80,45 @@ Thanks to all the contributors below for helping us making the documentation of
 | ------------------------------------ | ----------- | ------------------------------------------------------- |
 | [@author](https://github.com/author) | ...         | [#12345](https://github.com/nushell/nushell/pull/12345) |
 
+## Spread operator
+
+[#10598](https://github.com/nushell/nushell/issues/10598) asked for a spread operator (`...`), and [#11006](https://github.com/nushell/nushell/pull/11006)
```
Comment (amtoine) [inline blog/2023-12-12-nushell_0_88_0.md:85]: if you're ok with that, please add your name somewhere here, as a link to your GitHub account.
you are the one that made that possible, you should be in the spotlight with the spread operator next release :wink: :heart:

Comment (ysthakur) [reply blog/2023-12-12-nushell_0_88_0.md]: Always happy to toot my own horn :)

Comment (amtoine) [reply blog/2023-12-12-nushell_0_88_0.md]: perfect :yum:

**Pull request merged**